src/main/scala/com/adazza/example/spark/job/WordCount.scala

```diff
@@ -0,0 +1,36 @@
+package com.adazza.example.spark.job
+
+import org.apache.spark.sql.SparkSession
+import scala.concurrent.ExecutionContext
+
+/**
+  * Example Word Count Job that extends SparkJob trait
+  */
+object WordCount extends AdazzaSparkJob {
+  override def run(implicit ec: ExecutionContext,
```

Comment:
Don't know, but given the problems I ran into with running spark jobs in a threaded context, it probably shouldn't be. The "right" way to do parallelism in the cluster world is to invoke the jobs in parallel using an oozie workflow.